src/utils/config.js

```diff
@@ -76,6 +76,8 @@ const DEFAULTS = {
     blankColor: 'transparent'
   },
   BModal: {
+    // Defaults to the Boostrap default for the modal backdrop
```

Comment:
I would slightly change the comment to
```
// Defaults to the same value as the bootstrap Backdrop z-index
```

As the backdrops, modals containers, etc, all still have their bootstrap defined z-indexes, ans this z-index is used only for stacking order (as modals can be in-document anywhere, in any order)

The stacker DIV just creates a _new_ stacking context (where the content of the div retains it's stacking definition within that container. the content inside is stacked relative to each other, and then relative to the other stacker divs.

Once we move to portal-vue for modals (so that they all appended at the bottom of the document in a single container when open), the staking z-index trickery will go away, as the modals will always be in the portal-target in the order they will be stacked.